### PR TITLE
feat: Add configuration for stable environment

### PR DESCRIPTION
**Pull request by ds-lcapellino**
## Description

This PR will introduce environment specific configuration for the 24.08 release to run on the catena-x stable environment.

## Issue

relates to https://github.com/eclipse-tractusx/portal/issues/408

## Checklist

Please delete options that are not relevant.

- [x] I have performed a self-review of my changes
- [x] I have successfully tested my changes
- [x] I have added comments in the default values.yaml file with helm-docs syntax ('# -- ') if relevant for installation
- [x] I have commented my changes, particularly in hard-to-understand areas

### CODE REVIEW

File: environments/helm-values/values-stable.yaml [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+          - path: "/api/services"
+            pathType: "Prefix"
+            backend:
+              service: "services-service"
+              port: 8080
+
+  keycloak:
+    central:
+      clientId: "<path:portal/data/portal-iam-clientIds#central-client-id>"
+      clientSecret: "<path:portal/data.stable.iam/centralidp-client-secrets#reg-central-sa>"
```
Comment (evegufy) [inline environments/helm-values/values-stable.yaml:124]:
```suggestion
      clientSecret: "<path:portal/data/stable/iam/centralidp-client-secrets#reg-central-sa>"
```

**Review by evegufy — CHANGES_REQUESTED** [review]
Please fix secret path everywhere

File: environments/helm-values/values-stable.yaml (hunk 1 above)
Comment (ds-lcapellino) [reply environments/helm-values/values-stable.yaml]: Fixed paths

**Review by evegufy — APPROVED** [review]

**Comment by evegufy** [comment]
@ds-lcapellino as discussed, I'm closing this PR as it doesn't make sense to merge it main because there we're are already ahead of the v2.1.0, the branch can still be used for installing v2.1.0

**Pull request closed without merging**